### PR TITLE
Add ability to bypass isNpm check with `shouldNotifyInNpmScript` option

### LINKED ISSUE #122: Make the isNpm() check optional with a flag
I am currently working on a project that would still like the notification output when running in a npm script context.  Currently we have to make assumptions about how both update-notifier and isNpm are implemented and manually manipulate process.env to work around this issue.  This is not ideal.

I wanted to open a PR to add an notifyWithNpm flag to the notify options.  Only if explicitly set would the notification still be outputted in an npm context (not break current functionality).  Would this be something you all would be open to?

### PR DESCRIPTION
**Pull request by alexccl**
My team and I have run into multiple scenarios where we've wanted to output the update message when running as an npm script.  To do this, we've had to dig through the source code, hack environment variables, run this package, and reset environment variables.

This PR include functionality to bypass the isNpm() check.  This will not break any existing functionality.

---

Fixes #122

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

File: index.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ class UpdateNotifier {
 		this.callback = options.callback || (() => {});
 		this.disabled = 'NO_UPDATE_NOTIFIER' in process.env ||
 			process.argv.indexOf('--no-update-notifier') !== -1;
+		this.skipIsNpmCheck = options.shouldNotifyInNpmScript;
```
Comment (sindresorhus) [inline index.js]: Better to give it the same name in both places:

```js
this.shouldNotifyInNpmScript = options.shouldNotifyInNpmScript;
```

**Comment by Aaronius** [comment]
Is this waiting on something before being merged? I'd like to use it as well.

**Comment by SBoudrias** [comment]
I'll go ahead and merge/release this.

**Comment by PixleProgrambetaxl** [comment]
+1

**Pull request merged**